uuGroup.r

```diff
@@ -956,13 +986,13 @@ uuGroupUserAdd(*groupName, *user, *status, *message) {
                 if (*externalUser == "1") {
                         *http_code = ""
                         *message = ""
-                        rule_group_provision_external_user(*userName, $userNameClient, $rodsZoneClient, *http_code, *message);
+                        rule_group_provision_external_user(*userName, *creatorUser, *creatorZone, *http_code, *message);
                         if (*message != "") {
                                 writeLine("serverLog", "[EXTERNAL USER] *message");
                                 *status = *http_code;
                                 succeed; # Return here (fail would ruin the status and error message).
                         }
-                        writeLine("serverLog", "[EXTERNAL USER] User *userName added by $userNameClient on $rodsZoneClient.");
+                        writeLine("serverLog", "[EXTERNAL USER] User *userName added on the behalf of *creatorUser on *creatorZone.");
```

Comment:
Good idea to log the creatorUser and creatorZone here. Can you also include the $userNameClient / $rodsZoneClient in the log message, so that admins can look up in the logs who originally created a user (even if it was on behalf of someone else)?